amp/WEB-INF/src/org/digijava/module/aim/util/FeaturesUtil.java

```diff
@@ -1271,12 +1270,10 @@ public static boolean deleteTemplateVisibility(Long id) {
             }
         }
         catch (Exception ex) {
-            if (hbsession != null && hbsession.getTransaction().isActive()) {
-                hbsession.getTransaction().rollback();
-            }
             logger.error(ex.getMessage(), ex);
             return false;
-        } finally {
+        }
+        finally {
             if (hbsession != null) {
```

Comment:
you don even need to close the session that is taken care by the filter while committing